scenes/game/game.gd

```diff
@@ -1700,6 +1729,15 @@ func _on_reveal_hand(event):
 		pass
 	return SmallNoticeDelay
 
+func _on_reveal_random_gauge(event):
+	var player = event['event_player']
+	var card_id = event['number']
+	spawn_damage_popup("Random Gauge Card!", player)
+	
+	#TODO: would be nice to display the card briefly
```

Comment:
Will look into this; do you have a good reference for temporarily moving it over? It's only being "selected" from gauge so I wanted to avoid actually moving it out of the gauge logically, but maybe if it was just on a timer somehow

Reply:
It should be fine to call _move_card_to_strike_area.
I would kind of expect this to happen automatically as part of _on_strike_started, which looks like is actually happening on your EventType_Strike_RandomGaugeStrike.
I haven't fully analyzed the ordering of all of your events for this random gauge strike thing yet though, so i don't know exactly which events come when.

Reply:
Oh just to clarify, this particular function is for "Over Here" to try and give a visual of which gauge card was chosen for "X"; I think striking from gauge moves things to the strike area properly, but this effect is a bit weird because the chosen card doesn't ever actually leave gauge

Reply:
Oh, I see. In that case, I guess just popping up a notice with the X amount that was chosen would be appropriate?